utils/network.ts

```diff
@@ -15,9 +15,9 @@ export function isSupportedNetworkId(id: number | string): id is NetworkId {
 	return id in NetworkNameById;
 }
 
-export async function getDefaultNetworkId(): Promise<NetworkId> {
+export async function getDefaultNetworkId(walletConnected: boolean = true): Promise<NetworkId> {
 	try {
-		if (window.ethereum) {
+		if (walletConnected && window.ethereum) {
```

Comment:
only use specified network if user is connected -  this will [default to optimism](https://github.com/Kwenta/kwenta/pull/616/files#diff-1d57bb0118fc478a61a04d99d5b0d2405b1d01576521d8acb319ae2ff500243eR19) when not connected